### PR TITLE
Prerequisite

### PR DESCRIPTION
**Pull request by simonjenga**
Prerequisite

### CODE REVIEW

**Comment by davvd** [comment]
I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by davvd** [comment]
@krzyk this pull request is for you, please review

File: s3auth-relay/src/main/java/com/s3auth/relay/HttpRequest.java [hunk 1]
```diff
@@ -172,10 +172,7 @@
         final Collection<String> headers = new LinkedList<String>();
         while (true) {
             final String line = reader.readLine();
-            if (line == null) {
-                break;
-            }
-            if (line.isEmpty()) {
+            if (line == null || line.isEmpty()) {
```
Comment (krzyk) [inline s3auth-relay/src/main/java/com/s3auth/relay/HttpRequest.java]: it is better to use `StringUtils.isEmpty()` it has the check for null also

**Comment by krzyk** [comment]
@simonjenga one comment from me above

**Comment by simonjenga** [comment]
@krzyk You're right, I've now modified the validation according to your comment above

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor merge

**Comment by simonjenga** [comment]
Thanks all!

**Comment by davvd** [comment]
@krzyk thanks, I just added **16 mins** to your account, payment `53739864`; you're getting extra minutes here (c=1); +16 added to your rating, current score is: [+5034](http://www.netbout.com/b/35035?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**